README.md

```diff
@@ -266,7 +266,9 @@ By default, profiles are created with autoconnect enabled.
 The `mac` address is optional and restricts the profile to be usable only on
 devices with the given MAC address. `mac` is only allowed for `type`
 `ethernet` or `infiniband` to match a non-virtual device with the
-profile.
+profile. The value of `mac` address is string and should use with quotation mark.
+Because `mac` address value is separated by colon, if not quoted, YAML may parse it as
+a integer in sexagesimal(base 60) form. (For more details, check https://yaml.org/spec/1.1/#id858600)
```

Comment:
Thank you. There are a few grammar errors in the string and I realized that the description does not really tell that the address needs to use colons (sometimes hyphens are used, too). Here is a suggestion:

```suggestion
profile. The value of the `mac` address needs to be specified in hexadecimal notation using colons (for example: `mac: "00:00:5e:00:53:5d"`). To avoid YAML parsing mac addresses as integers in sexagesimal (base 60) notation (see https://yaml.org/spec/1.1/#id858600), it is recommended to always quote the value with double quotes and sometimes it is necessary.
```

Reply:
updated